testfiles/gedcom70/lang-all.ged

```diff
@@ -0,0 +1,112 @@
+﻿0 HEAD
+1 SOUR TEST_FILES
+1 SUBM @1@
+1 GEDC
+2 VERS 7.0
+1 LANG af
+1 SCHMA
+2 TAG _PHRASE https://gedcom.io/terms/v7/PHRASE
+0 @1@ SUBM
+1 NAME Luther
+1 LANG en
+1 LANG ja
+1 LANG es
+0 @2@ _ALL_LANGUAGES
```

Comment:
Because this is an extension tag, its substructures aren't obviously lanugages. Also, the name is misleading.

```suggestion
0 @2@ SUBM
1 NAME GEDCOM 5.5.1
1 NOTE This contains the language tags for every language named in the v5.5.1 spec, namely: Afrikaans, Albanian, Amharic, Anglo-Saxon, Arabic, Armenian, Assamese, Belorusian, Bengali, Braj, Bulgarian, Burmese, Cantonese, Catalan, Catalan_Spn, Church-Slavic, Czech, Danish, Dogri, Dutch, English, Esperanto, Estonian, Faroese, Finnish, French, Georgian, German, Greek, Gujarati, Hawaiian, Hebrew, Hindi, Hungarian, Icelandic, Indonesian, Italian, Japanese, Kannada, Khmer, Konkani, Korean, Lahnda, Lao, Latvian, Lithuanian, Macedonian, Maithili, Malayalam, Mandrin, Manipuri, Marathi, Mewari, Navaho, Nepali, Norwegian, Oriya, Pahari, Pali, Panjabi, Persian, Polish, Portuguese, Prakrit, Pusto, Rajasthani, Romanian, Russian, Sanskrit, Serb, Serbo_Croa, Slovak, Slovene, Spanish, Swedish, Tagalog, Tamil, Telugu, Thai, Tibetan, Turkish, Ukrainian, Urdu, Vietnamese, Wendic, Yiddish```